datahub-web-react/src/app/search/utils/constants.ts

```diff
@@ -94,3 +94,5 @@ export enum UnionType {
     AND,
     OR,
 }
+
+export const SCROLL_KEEP_ALIVE_TIME = '10m'; // TODO: Determine if this is sufficient.
```

Comment:
Can we make this an environment variable instead?

Reply:
Environment variable? That means extending a config endpoint right?

Reply:
Kinda blows up scope to adding this to our appConfig GraphQL API. Let me confirm with Ryan on platform team whether this is something we'd need to allow config on